webapp/client/src/components/FormNavButtons.js

```diff
@@ -92,42 +93,48 @@ const ExplanatoryText = styled.small`
 
 export default function FormNavButtons({
   explanatoryButtonText,
-  overviewButton,
+  showOverviewButton,
   nextButtonLabel,
 }) {
-  // TODO intl
-  const backToOverviewText = "form.back_to_overview";
-  // TODO intl
-  nextButtonLabel = nextButtonLabel || "form.next";
+  const { t } = useTranslation();
```

Comment:
What's that syntax?! 😳

Reply:
Destructuring assignment. It's beautiful ❤️. https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment